images/wkdev_sdk/jhbuild/webkit-sdk-deps.modules

```diff
@@ -99,7 +99,11 @@
     <branch repo="gstreamer.freedesktop.org"
             checkoutdir="gstreamer"
             module="gstreamer.git"
-            tag="1.24.6"/>
+            tag="1.24.6">
+      <patch file="0001-webrtcbin-create-and-associate-transceivers-earlier-.patch" strip="1"/>
+      <patch file="0002-webrtcbin-reverse-direction-from-remote-media.patch" strip="1"/>
+      <patch file="0003-webrtcbin-connect-output-stream-on-recv-transceivers.patch" strip="1"/>
```

Comment:
Patches don't apply to git sources. So this would have to either use a tarball or change the commit to one that includes these.

Reply:
The former then, because the latter is not possible (we shouldn't use git main imho). Thanks for the review! I'll update the PR tomorrow.

Reply:
Oh but then we'd need to split modules again (-base, -good, -bad, etc), huh... sigh :(

Reply:
> Patches don't apply to git sources

Whyyyy?

Reply:
Its unfortunate, I'm not sure the history behind it. One day I'd like to fix that in jhbuild.

Reply:
https://gitlab.gnome.org/GNOME/jhbuild/-/merge_requests/257 r?

Reply:
This was merged so this can be rebuilt and tested.